docs/content/en/install/doks/_index.md

```diff
@@ -0,0 +1,332 @@
+```
+
+**Output**:
+
+```bash
+secret/prod-k8ssandra-medusa-key created
+```
+
+{{% alert title="Important" color="primary" %}}
+The name of the key file within the secret MUST be `medusa_s3_credentials`. _Any_ other value will result in Medusa not finding the secret and backups failing.
+{{% /alert %}}
+
+This secret, `prod-k8ssandra-medusa-key`, can now be referenced in our K8ssandra configuration to allow for backing up data to DigitalOcean Spaces with Medusa.
+
+### Create `WaitForFirstConsumer` Storage Class
+
+With K8ssandra it is recommended that a Kubernetes Storage Class with `volumeBindingMode: WaitForFirstConsumer`. By default the pre-installed `do-block-storage` storage class uses `Immediate`. We will create a new storage class with this parameter changed for use with K8ssandra. First retrieve the existing storage class:
+
+```console
+kubectl get sc do-block-storage --output yaml | tee do-block-storage-wait.yaml
+```
+
+**Output:**
+
+```console
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"allowVolumeExpansion":true,"apiVersion":"storage.k8s.io/v1","kind":"StorageClass","metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"true"},"name":"do-block-storage"},"provisioner":"dobs.csi.digitalocean.com","reclaimPolicy":"Delete"}
+    storageclass.kubernetes.io/is-default-class: "true"
+  creationTimestamp: "2021-06-08T15:24:10Z"
+  name: do-block-storage
+  resourceVersion: "315"
+  uid: 72005420-ae87-4d3a-a3ed-6022bf6cb297
+provisioner: dobs.csi.digitalocean.com
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+```
+
+Next we edit this file removing all of the metadata (except `name`) and adjust `volumeBindingMode` to `WaitForFirstConsumer`. Your updated storage class should look like the following:
+
+{{< readfilerel file="do-block-storage-wait.yaml" highlight="yaml" >}}
+
+Submit this file to the Kubernetes cluster with:
+
+```console
+kubectl apply -f do-block-storage-wait.yaml 
+```
+
+**Output:**
+
+```console
+storageclass.storage.k8s.io/do-block-storage-wait created
+```
+
+### Generate `doks.values.yaml`
+
+Here is a reference Helm `values.yaml` file with configuration options for running K8ssandra in DigitalOcean on DOKS.
+
+{{< readfilerel file="doks.values.yaml"  highlight="yaml" >}}
+
+{{% alert title="Important" color="primary" %}}
+Take note of the comments in this file. If you have changed the name of your secret, are deploying in a different region, or have tweaked any other values it is imperative that you update this file before proceeding.
+{{% /alert %}}
+
+### Deploy K8ssandra with Helm
+
+With a `values.yaml` file generated which details out specific configuration overrides we can now deploy K8ssandra via Helm.
+
+```bash
+helm install prod-k8ssandra k8ssandra/k8ssandra -f doks.values.yaml
```

Comment:
above you refer to the file as `values.yaml`, now in this command it is called `docs.values.yaml`. Should be consistent.